packages/lodestar/src/sync/initial/fast/index.ts

```diff
@@ -220,18 +221,10 @@ export class FastSync extends (EventEmitter as {new (): InitialSyncEventEmitter}
    * Returns peers which has same finalized Checkpoint
    */
   private getInitialSyncPeers = async (): Promise<PeerId[]> => {
-    return this.network
-      .getPeers({
-        connected: true,
-        supportsProtocols: getSyncProtocols(),
-      })
-      .map((peer) => peer.id)
-      .reduce((validPeers: PeerId[], peer: PeerId) => {
-        const status = this.network.peerMetadata.getStatus(peer);
-        if (status && status.finalizedEpoch >= this.targetCheckpoint!.epoch) {
-          validPeers.push(peer);
-        }
-        return validPeers;
-      }, [] as PeerId[]);
+    const targetCheckpoint = this.targetCheckpoint;
+    return getSyncPeers(this.network, (peer) => {
```

Comment:
do we want to apply this to regular sync `naive.ts` strategy to? probably in `checkBestPeer()`

Reply:
yeah, makes sense